2021_JuMP/slides/marek.tex

```diff
@@ -0,0 +1,201 @@
+\begin{frame}{Group Theory to the rescue!}
+  \begin{block}{Definition}
+    A \textbf{group} $(G, *)$ is a set closed under the multiplication $*$, together with the existence of unique inverse elements.
+  \end{block}
+    {\small Example: The set of all bijections of the set $\{1,\ldots n\}$ with composition is a group (known as the full symmetric group on $n$ letters $S_n$).}
+    
+    $S_2 = \{(1)(2), (1,2)\}$ \textbf{acts} on polynomial ring $\mathbb{R}[x]$ by
+    \begin{align*}
+      ((1)(2), x) & \mapsto x\\
+      ((1, 2), x) & \mapsto -x.
+    \end{align*}
+    {\small Observe: $f(x) = x^4 - 2x^2$ is invariant under this action.\\
+    Observe: $f(x) = x^6 - 2x^4 + 2x^2 = x^2 + (x-x^3)^2$ is invariant, but one of the summands isn't!}
+    
+\end{frame}
+
+
+\begin{frame}{Isotypical projections}
+    ``Types'' of actions $\leftrightarrow$  blocks in the block-diagonalization:
+    \[V \cong V_1 \oplus \cdots \oplus V_k.\]
+    
+    Each block $V_i$ decomposes further as 
+    $V_i = \oplus_{m_i} V_i'$. (\footnote{$m_i$ is the multiplicity of the action type in $V$})
```

Comment:
"in $V$" -> "in $V_i$" ?

Reply:
that's the same, but since we have 5 more days, I'll rework this part anyway ;)